ui/assets/excluded_files.tpl.html

```diff
@@ -9,8 +9,11 @@
     <body>
         <div id="root"></div>
 
-        <script src="js/react-{{.ReactVersion}}.min.js"></script>
+        <script src="js/react-with-addons-{{.ReactVersion}}.min.js"></script>
+        <script src="js/react-dom-{{.ReactVersion}}.min.js"></script>
         <script src="js/jquery-{{.jQueryVersion}}.min.js"></script>
+        <script src="js/babel-core.5.8.23.browser.min.js"></script>
+        <script src="js/common.js"></script>
```

Comment:
The babel core is replacing the JSX inline transpiler. This can be skipped entirely by transpiling the JSX and only sending that. 

We could also probably combine React with addons and React Dom together in one file if necessary.